daemon/snap.go

```diff
@@ -183,19 +179,22 @@ func mapLocal(about aboutSnap) map[string]interface{} {
 		appNames = append(appNames, appName)
 	}
 	sort.Strings(appNames)
-	apps := make([]appJSON, 0, len(localSnap.Apps))
-	for _, appName := range appNames {
+	apps := make([]*client.AppInfo, len(localSnap.Apps))
```

Comment:
So is it okay to now pull in client structures in the daemon? If so maybe I could move some interface bits there instead?